mm-go-irckit/service.go

```diff
@@ -102,41 +111,55 @@ func login(u *User, toUser *User, args []string, service string) {
 		datalen--
 	}
 
-	if len(args) == datalen {
-		cred.Pass = args[len(args)-1]
-		cred.Login = args[len(args)-2]
+	if len(args) >= datalen {
```

Comment:
I don't get why it's failing here. We had `if len(args) == datalen` at the same position before?

Reply:
It's failing because the original code hasn't been linted with the nestif linter, new code/PR's will.
In this case the burden of refactoring this issue shouldn't be on you.

```suggestion
	if len(args) >= datalen { // nolint:nestif
```